RELEASE_NOTES.md

```diff
@@ -1,5 +1,156 @@
 # Release Notes
 
+## Changes in Splinter 0.5.19
+
+### Highlights
+
+* The experimental `metrics` feature has been renamed to `tap` and stabilized.
+
+* The use of PeerAuthorizationToken as the Peer ID has been replaced by
+  `PeerTokenPair` which is made up of both the other nodes authorization type
+  and the local authorization type used for the local node.
+
+* Simply back-pressure has been added to the scabbard `/batches` route. If
+  there are more than 30 pending batches, the scabbard route will start
+  rejecting the batch and returning TOO_MANY_REQUESTS, until the batch queue
+  reduces by half.
+
+### libsplinter
+
+* Update `metrics` feature to use v0.17 of metrics in libsplinter. Due to
+  change in v0.17 the `metrics` feature was renamed to `tap` to avoid
+  conflicts.
+
+*  Make `InfluxRecorder::new` private, `InfluxRecorder::init` should be used
+   instead.  The `InfluxRecorder` is available behind the `tap` feature.
+
+* Remove clones from `InfluxRecorder`. This change removes clones from the
+  influx recorder, reducing the clones of entire strings. While small, metrics
+  will happen frequently as more metrics are added, so its performance impact
+  should be as minimal as possible.  The `InfluxRecorder` is available behind
+  the `tap` feature.
+
+* Add connection_id to `ConnectionManagerNotification::Disconnected` and
+  `ConnectionManagerNotification::NonFatalConnectionError`. This will help with
+  switching peers to be uniquely identified by the identity and the local
+  identity used.
+
+* Add picked public key to `AuthChallengeSubmitResponse`. This will allow the
+node to know which public key is being used as their identity on the other node
```

Comment:
Please update the doc to the formatting is consistent, some sections do not line up with the others.